src/compiler/config/outputs/validate-dist.ts

```diff
@@ -11,7 +8,10 @@ import {
   DIST_TYPES,
   getComponentsDtsTypesFilePath,
   isOutputTargetDist,
-} from '../../output-targets/output-utils';
+} from '@utils';
```

Comment:
Can we combine this with the existing `@utils` import?

Reply:
oops, thought I captured all of those!